apps/frontend/src/hooks/use-input.ts

```diff
@@ -1,13 +1,26 @@
 import { useState } from 'react';
+import { InputType } from '../utilities/constants';
 
-const useInput = (validateValue) => {
+const useInput = (validateValue, inputType: InputType = InputType.ORIGINAL) => {
   const [enteredValue, setEnteredValue] = useState('');
   const [isTouched, setIsTouched] = useState(false);
 
+  const normalizeValue = (value: string) => {
```

Comment:
- We can directly convert the invoice/offer value to lowercase as done in the initial PR. However, converting the `useInput` hook to accept a type will make the project and other inputs more future-ready.

- Adding a basic set of automated tests could enhance the overall quality of the PR.